microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,175 @@
+
+To choose a different trace type when exporting:
+
+1. Right-click (or hold the `Ctrl` key and click) the **Save profile** (![Save profile button](../media/export-icon.png)) in the **Performance** tool or the **Save** button in the **Memory** tool.
+
+1. Click either **.devtools (enhanced format for Microsoft Edge)** if you want to export an enhanced trace, or click **.json (works with Microsoft Edge and Chromium browsers)** if you want a normal trace.
+
+   ![The Performance tool, showing the contextual menu on the Save profile button, which allows to choose the type of trace](./share-traces-images/choose-trace-type-on-export.png)
+
+
+<!-- ====================================================================== -->
+## Import a trace in the Performance tool
+
+To import a trace in the **Performance** tool:
+
+1. Open the **Performance** tool.
+
+1. Click **Load profile** (![Load profile button](../media/import-icon.png)):
+
+   ![Microsoft Edge, showing the Performance tool in DevTools, with the Load profile button](./share-traces-images/importing-perf-trace.png)
+
+1. Locate the trace file on your disk. It's either a `.devtools` file (for enhanced traces) or a `.json` file (for normal traces):
+
+   ![The Windows open dialog, showing a performance trace file](./share-traces-images/opening-perf-trace.png)
+
+1. Open the file.
+
+   If the file is an enhanced trace, a new DevTools window appears, showing the performance profile and the extra runtime information that was recorded displayed in the **Sources**, **Console**, and **Elements** tools.
+
+   If the file is a normal trace, the performance profile appears in the **Performance** tool, and the rest of the DevTools tabs continue to show information related to the current webpage.
+
+
+<!-- ====================================================================== -->
+## Import a trace in the Memory tool
+
+To import a trace in the **Memory** tool:
+
+1. Open the **Memory** tool.
+
+1. Click **Load**.
+
+   ![Microsoft Edge, showing the Memory tool in DevTools, with the Load button](./share-traces-images/importing-memory-trace.png)
+
+1. Locate the trace file on your disk. It's either a `.devtools` file (for enhanced traces) or a `.heapsnapshot`, `.heaptimeline`, or `.heapprofile` file (for normal traces).
+
+   ![The Windows open dialog, showing a memory trace file](./share-traces-images/opening-memory-trace.png)
+
+1. Open the file.
+
+   If the file is an enhanced trace, a new DevTools window appears, showing the memory information and the extra runtime information that was recorded displayed in the **Sources**, **Console**, and **Elements** tools.
+
+   If the file is a normal trace, the memory information appears in the **Memory** tool, and the rest of the DevTools tabs continue to show information related to the current webpage.
```

Comment:
Can we add something at the end, maybe just like: 

This is an experimental feature that hopefully makes it easier to collaborate with DevTools and see unminified file and function names in the Performance and Memory tools. If you are driving memory and performance investigations, please give this feature a try and let us know how it goes for you by filing feedback [in our GitHub repo](https://github.com/microsoftedge/DevTools/)!

Maybe consider creating a default issue we can get feedback for in the GitHub repo. I'll defer to you on what's best here.